1zadanie.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="1zadanie.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:200,300,300i,400,400i,700,700i&amp;subset=latin-ext" rel="stylesheet">
+    
+    </head>
+<body>
+
+    
+     <section id="layout" >
+         <img src="coming-soon-ribbon.png" id="coming_soon">
+         <img src="undraw_developer_activity_bv83.svg" id="guy" >
+         <div id="code" style="color: white;">CODE ACADEMY</div>
+         <h3>Leave your email</h3>
```

Comment:
A H1 i H2 gdzie są?

Reply:
tak się z ciekawości spytam, a muszą być? ;> nie można sobie użyć h3 jak nigdzie nie było h1, ani h2? przecież to pełnoprawny element HTML'a

Reply:
@mateuszlittwin-gf, jeśli masz gdzieś semantykę, to nie muszą ;) Najlepiej jest dać tytuł główny dokumentu w H1, H2 jako tytuły sekcji, H3 jako artykułów itd ;) Choćby ze względu na czytelność kodu i stworzenie swoistego "drzewka" strony.

Reply:
to tym bardziej tutaj nie pasuje h1 ani h2, bo to też nie są tytuły sekcji ;) bardziej tu pasuje `p`

Reply:
Myślę, że użycie nagłówka tutaj jest do przyjęcia. Choćby ze względu na fakt, że element powinien zwrócić uwagę odwiedzającego. Jednak po drodze zabrakło właśnie header tagów poprzednich ;)

Reply:
Teraz tak patrze szerzej i `coming-soon-ribbon` mozna wywalic gdzies wyzej, no on chyba i tak jest z position absolute (zgaduje).
Ten obrazek `undraw_developer_activity` pasuje bardziej do jakiegos znacznika header, albo innej sekcji
Pozostałe elementy faktycznie mogą miec `h1` i `h2`, czyli
``` html
<h1>Code Academy</h1>
<h2>Leave you email</h2>
```

Reply:
O coś takiego mi dokładnie chodzi ;) Gdyby dodatkowo ładnie rozdzielić plik na sekcje header, main i footer byłoby niemal idealnie :)